src/locales/ko/mystery-encounters/teleporting-hijinks-dialogue.json

```
@@ -1,27 +1,27 @@
 {
-  "intro": "It's a strange machine, whirring noisily...",
-  "title": "Teleportating Hijinks",
-  "description": "The machine has a sign on it that reads:\n  \"To use, insert money then step into the capsule.\"\n\nPerhaps it can transport you somewhere...",
-  "query": "What will you do?",
+  "intro": "시끄럽게 윙윙 소리가 나는 이상한 기계를 발견했다….",
+  "title": "텔레포트 대소동",
+  "description": "기계에는 다음과 같은 표지판이 있습니다:\n  \"사용하려면, 돈을 넣고 캡슐에 들어가세요.\"\n\n어딘가로 당신을 데려다줄 수 있을겁니다…….",
+  "query": "어떻게 하시겠습니까?",
   "option": {
     "1": {
-      "label": "Put Money In",
-      "tooltip": "(-) Pay {{price, money}}\n(?) Teleport to New Biome",
-      "selected": "You insert some money, and the capsule opens.\nYou step inside..."
+      "label": "돈을 넣는다",
+      "tooltip": "(-) {{price, money}} 지불\n(?) 다른 바이옴으로 이동",
+      "selected": "돈을 넣자, 캡슐이 열렸다.\n당신은 안으로 들어갔다……."
     },
     "2": {
-      "label": "A Pokémon Helps",
-      "tooltip": "(-) {{option2PrimaryName}} Helps\n(+) {{option2PrimaryName}} gains EXP\n(?) Teleport to New Biome",
-      "disabled_tooltip": "You need a Steel or Electric Type Pokémon to choose this",
-      "selected": "{{option2PrimaryName}}'s Type allows it to bypass the machine's paywall!$The capsule opens, and you step inside..."
+      "label": "포켓몬의 도움받기",
+      "tooltip": "(-) {{option2PrimaryName}}[[가]] 도와줌\n(+) {{option2PrimaryName}}[[가]] 경험치를 얻음\n(?) 다른 바이옴으로 이동",
+      "disabled_tooltip": "강철 또는 전기 타입 포켓몬이 있어야한다.",
+      "selected": "{{option2PrimaryName}}의 타입으로 기계의 지불 시스템을 해제했다!$캡슐이 열리자, 당신은 안으로 들어갔다…."
     },
     "3": {
-      "label": "Inspect the Machine",
-      "tooltip": "(-) Pokémon Battle",
-      "selected": "You are drawn in by the blinking lights\nand strange noises coming from the machine...$You don't even notice as a wild\nPokémon sneaks up and ambushes you!"
+      "label": "기계를 조사한다",
+      "tooltip": "(-) 포켓몬 배틀",
+      "selected": "기계에서 나오는 깜빡이는 불빛과\n이상한 소리를 푹 빠져서 조사했다…….$정신이 팔려서 야생 포켓몬이\n다가오는 것도 눈치채지 못했다!"
     }
   },
-  "transport": "The machine shakes violently,\nmaking all sorts of strange noises!$Just as soon as it had started, it quiets once more.",
-  "attacked": "You step out into a completely new area, startling a wild Pokémon!$The wild Pokémon attacks!",
-  "boss_enraged": "The opposing {{enemyPokemon}} has become enraged!"
+  "transport": "기계가 심하게 흔들리면서,\n온갖 이상한 소리가 나기 시작했다!$그리고 텔레포트를 시작하자마자, 다시 조용해졌다.",
+  "attacked": "새 바이옴에 도착하자마자, 거기에 있던 야생 포켓몬을 놀래켰다!$야생 포켓몬이 공격해온다!",
```

```suggestion
  "attacked": "새로운 지역에 발을 내딛자마자 그곳의 야생 포켓몬을 놀라게 했다!$야생 포켓몬이 공격해 왔다!",
```
1) 저도 몰랐는데 방금 사전을 찾아보니 '놀래키다'는 표준어가 아니라 사투리라고 나오네요... 표준어는 '놀라게 하다'='놀래다'라고 합니다
2) 사실 바이옴이긴 한데 원문에는 그냥 area라고 되어 있으니 그걸 살리는 게 좋지 않을까 제안합니다.